test-utils/src/helpers.rs

```diff
@@ -64,6 +64,24 @@ pub fn internal_error(id: Id) -> String {
 	)
 }
 
+/// Hardcoded server response when a client initiates a new subscription.
+///
+/// NOTE: works only for one subscription because the subscription ID is hardcoded.
+pub fn server_subscription_id_response(id: Id) -> String {
+	format!(
+		r#"{{"jsonrpc":"2.0","result":"D3wwzU6vvoUUYehv4qoFzq42DZnLoAETeFzeyk8swH4o","id":{}}}"#,
+		serde_json::to_string(&id).unwrap()
+	)
+}
+
+/// Server response to a hardcoded pending subscription
+pub fn server_subscription_response(result: Value) -> String {
```

Comment:
Can these functions be `const` these days?

Reply:
https://docs.serde.rs/serde_json/fn.to_string.html is not const and I think `format!` doesn't work in const context.